lib/src/command/remove.dart

```diff
@@ -20,9 +20,19 @@ class RemoveCommand extends PubCommand {
   @override
   String get name => 'remove';
   @override
-  String get description => 'Removes a dependency from the current package.';
+  String get description => '''
+Removes dependencies from `pubspec.yaml`.
+
+Invoking `dart pub remove foo bar` will remove `foo` and `bar` from either
+`dependencies` or `dev_dependencies` in `pubspec.yaml`.
+
+To remove a dependency override of a package prefix the package name with
+'override:'.
+''';
+
   @override
-  String get argumentsDescription => '<package>';
+  String get argumentsDescription =>
+      '[override:]<package1> [[override:]<package2>...]';
```

Comment:
Should we consider:
```suggestion
  String get argumentsDescription => '<package1> [<package2>...]';
```
To keep the usage line simple. I guess it's wrong, I'm just tempted to :D